lib/please_cli/please_cli/build.py

```diff
@@ -143,5 +143,94 @@ def cmd(project,
         please_cli.utils.check_result(result, output)
 
 
+@click.command()
+@cli_common.click.taskcluster_options
+@click.argument(
+    'project',
+    required=True,
+    type=click.Choice(please_cli.config.PROJECTS),
+    )
+@click.option(
+    '--nix-build',
+    required=True,
+    default=please_cli.config.NIX_BIN_DIR + 'nix-build',
+    help='`nix-build` command',
+    )
+@click.option(
+    '--docker',
+    required=True,
+    default='/usr/bin/docker',
+    help='Path to docker binary on your host',
+    )
+@click.option(
+    '--interactive/--no-interactive',
+    default=True,
+    )
+@click.option(
+    '--load-image/--no-load-image',
+    help='Load the generated image into docker',
+    default=True,
+    )
+def cmd_docker(project,
+        nix_build,
+        taskcluster_secret,
+        taskcluster_client_id,
+        taskcluster_access_token,
+        docker,
+        interactive,
+        load_image,
+    ):
+
+    docker_path = please_cli.config.TMP_DIR + '/result-docker-{project}'.format(project=project)
```

Comment:
Nit: call this `image_path` or something like this, to avoid confusing it with the `docker` variable which contains the path to Docker.